### DISCUSSION TITLE
Markdoc Language Server: Auto complete for attributes.

### DISCUSSION THREAD

**Discussion by louiss0** [markdoc/markdoc#431] [Ideas]
I see that right now I can't seem to be able to press ctrl space to get suggestions for attributes.

I wish that was a thing. When it comes to tags I do receive it but I don't get it for attributes. Most attributes will never be required.
They will be optional. The user should not have to keep reading the docs to get information on what attributes exist.

**Comment by louiss0** [comment]
Here is a repo you can use to test out this functionality. It uses the [Markdoc HTML Tag Schemas Library](https://markdoc-html-tag-schemas-docs.onrender.com) and is properly configured. Here the link to the project https://github.com/louiss0/dummy-markdoc-2

**Comment by louiss0** [comment]
It still does not work. I did update the extension.

**Comment by louiss0** [comment]
The IntelliSense is not working for attributes.  
![demonstration-of-markdoc-not-giving-autocomplete-for-attrs](https://github.com/markdoc/markdoc/assets/29028601/c23b1ef1-93c8-4ee2-bda8-e0393e4ebbdf)

**Reply by branberry** [reply]
This would be great to have working. Trying to configure this with Astro, but no luck
